app/controllers/api/v2/auth_source_ldaps_controller.rb

```diff
@@ -45,7 +45,7 @@ def show
           param :usergroup_sync, :bool, :desc => N_("sync external user groups on login")
           param :tls, :bool
           param :groups_base, String, :desc => N_("groups base DN")
-          param :use_netgroups, :bool, :desc => N_("use NIS netgroups instead of posix groups, applicable only when server_type is posix or free_ipa")
+          param :ldap_group_membership, AuthSourceLdap::GROUP_MEMBERSHIP_TYPES.keys, :desc => N_("type of group membership to use, applicable only when server_type is posix, free_ipa or netiq. Option rfc4519 is only applicable  when server_type is posix.")
```

Comment:
This breaks the API (intentionally), but I wonder if we could (should?) add a transitional parameter `use_netgroups` that will translate internally into `ldap_group_membership = nis_netgroups`.
This will allow people with older Ansible modules to still use NIS.

Reply:
Partially. It would always try to update the auth source (so, pretending to be non-idempotent), but it'd "mostly work"
If you add a fake `use_netgroups` to the `json.rabl`, then it would not be confused at all.